themes/classic/assets/css/theme.css

```diff
@@ -4823,7 +4818,11 @@ ul {
     margin-top: 1rem; }
   section.checkout-step {
     width: 100%; }
-  section.checkout-step input[name=address1], section.checkout-step input[name=address2], section.checkout-step input[type=text], section.checkout-step input[type=password], input[type=date] {
+  section.checkout-step input[name=address1],
+  section.checkout-step input[name=address2],
+  section.checkout-step input[type=text],
+  section.checkout-step input[type=password],
```

Comment:
The name attributes are used into the `.php` files, so it can't be modified without overrides. If the module developer overrides it, he will be able to add its own `.css` files